new: custom hit error bar by breadles5

breadles5 opened the pull request:
# compatibility

tosu only

# previews

![customhiterrorbar.png](https://github.com/breadles5/customHitErrorBar/blob/master/.github/assets/customhiterrorbar.png?raw=true)  
![customhiterrorbar.gif](https://github.com/breadles5/customHitErrorBar/blob/master/.github/assets/customhiterrorbar.gif?raw=true)

cyperdark commented on counters/customhiterrorbar by breadles5/main.js:
```diff
@@ -0,0 +1,658 @@
+  firstObjectTime: 0,
+  isReset: true
+};
+const DEFAULT_HOST = "127.0.0.1:24050";
```
use `window.location.host` instead

breadles5 replied:
got it.

cyperdark commented on counters/customhiterrorbar by breadles5/main.js:
```diff
@@ -0,0 +1,658 @@
+        console.error("[SETTINGS] Error:", message.error);
+        return;
+      }
+      const newHost = message?.websocketUrl;
```
you dont have `websocketUrl` setting

breadles5 replied:
![image](https://github.com/user-attachments/assets/bab8db1d-7bb3-48d6-ae61-6d053a6dded5)
![image](https://github.com/user-attachments/assets/6b6b463e-d9ed-4e72-b3ce-ba99de235750)

cyperdark replied:
if you dont have setting for websocketUrl then no need to add it

breadles5 commented:
i adjusted the websocket.commands(...) and removed the unnecessary WebSocketCommandMessage and pushed those changes into this branch's main.js, so hopefully it should be good now
